### DISCUSSION TITLE
ChatGPT plus

### DISCUSSION THREAD

**Discussion by oscar091466** [GorvGoyl/ChatGPT-Writer#51] [Report Bugs and Issues]
Stopped working after i signed up for Chatgpt Plus 

getting this message 

There was an unknown error. Make sure you are logged-in to [chat.openai.com](https://chat.openai.com/)
If issue persists, please [report it here](https://github.com/GorvGoyl/ChatGPT-Writer/discussions/categories/report-bugs-and-issues).

Made sure I was logged in (Edit)

Currently on Chrome using Gmail.

**Comment by oscar091466** [comment]
Resolved. I uninstalled and reinstalled the extension

**Reply by AlRFORCE1** [reply]
This did not work for me =/

**Reply by rajeshkandel2023** [reply]
This did not work for me.

**Comment by ormanbeckles** [comment]
I also have chat GPT plus tried unloading and reinstalling the extension but still getting the same error

**Reply by oscar091466** [reply]
I created a regular account. uninstalled and reinstalled the extension and it worked. logged out of the free account and back to GPT plus and it was working again. not sure which step fixed it.

**Reply by rajeshkandel2023** [reply]
This did not work for me.

**Comment by GorvGoyl** [comment]
Thanks for reporting. I've fixed the issue in v2.1.9. It should be live within a day and then you can update the extension or restart the browser.

https://chrome.google.com/webstore/detail/chatgpt-writer-email-writ/pdnenlnelpdomajfejgapbdpmjkfpjkp/

**Comment by rajeshkandel2023** [comment]
Hi GorvGoyl/ChatGPT-Writer,

It is working now.

Thank you for fixing it promptly.
